packages/backend-core/src/environment.ts

```diff
@@ -154,13 +161,14 @@ const environment = {
   ENABLE_SSO_MAINTENANCE_MODE: selfHosted
     ? process.env.ENABLE_SSO_MAINTENANCE_MODE
     : false,
-  VERSION: findVersion(),
+  ...getPackageJsonFields(),
   DISABLE_PINO_LOGGER: process.env.DISABLE_PINO_LOGGER,
   _set(key: any, value: any) {
     process.env[key] = value
     // @ts-ignore
     environment[key] = value
   },
+  ROLLING_LOG_MAX_SIZE: process.env.ROLLING_LOG_MAX_SIZE || "100M",
```

Comment:
is this 100M of log by default? I think we should reduce the size of this a bit for self hosters - thats quite a large log file (for uploading etc as well)

Reply:
What do you think would be a good size? 20MB?

Reply:
I think `10MB` would be good!